src/ui/MentionLabel/index.tsx

```diff
@@ -0,0 +1,88 @@
+import './index.scss';
+
+import React, {
+  ReactElement,
+  useRef,
+  useState,
+  useCallback,
+} from 'react';
+import type SendBird from 'sendbird';
+
+import ContextMenu, { MenuItems } from '../ContextMenu';
+import Label, { LabelTypography, LabelColors } from '../Label';
+import { UserProfileContext } from '../../lib/UserProfileContext';
+import UserProfile from '../UserProfile';
+import useSendbirdStateContext from '../../hooks/useSendbirdStateContext';
+
+interface MentionLabelProps {
+  mentionTemplate: string;
+  mentionedUserId: string;
+  isByMe: boolean;
+}
+
+export default function MentionLabel(props: MentionLabelProps): JSX.Element {
+  const {
+    mentionTemplate,
+    mentionedUserId,
+    isByMe,
+  } = props;
+
+  const mentionRef = useRef();
+
+  const sendbirdState = useSendbirdStateContext();
+  const userId = sendbirdState?.config?.userId;
+  const sdk = sendbirdState?.stores?.sdkStore?.sdk;
+  const iAmBeingMentioned = userId === mentionedUserId;
+  const [user, setUser] = useState<SendBird.User| null>();
+  const fetchUser = useCallback(
+    (toggleDropdown) => {
+      if (user) {
+        toggleDropdown();
+      }
+      const query = sdk.createApplicationUserListQuery();
+      query.userIdsFilter = [mentionedUserId];
+      query.next((members) => {
+        if (members?.length > 0) {
+          setUser(members[0]);
+        }
+        toggleDropdown();
+      })
+    },
+    [sdk, mentionedUserId],
+  );
+  return (
+    <ContextMenu
+      menuTrigger={(toggleDropdown: () => void): ReactElement => (
+        <a
+          className={`
+            sendbird-word__mention
+            ${iAmBeingMentioned && 'sendbird-word__mention--me'}
```

Comment:
How about this
```
${iAmBeingMentioned ? 'sendbird-word__mention--me' : ''}
```
because above returns 'false' as string type

Reply:
yes will do 👍